abc.templates/deployments/contents/deploy.sh

```diff
@@ -3,20 +3,21 @@ set -euo pipefail
 
 GITHUB_METRICS_VERSION='REPLACE_GITHUB_METRICS_VERSION_TAG'
 PLATFORM='amd64'
+VERSION="${GITHUB_METRICS_VERSION}-${PLATFORM}"
+UPSTREAM_IMAGE_NAME="us-docker.pkg.dev/abcxyz-artifacts/docker-images/github-metrics-aggregator:${VERSION}"
 IMAGE_NAME="REPLACE_FULL_IMAGE_NAME:${GITHUB_METRICS_VERSION}-${GITHUB_SHA}"
 
-docker build -t "${IMAGE_NAME}" \
-  --build-arg="VERSION=${GITHUB_METRICS_VERSION}-${PLATFORM}" \
-  REPLACE_SUBDIRECTORY
-
-docker push "${IMAGE_NAME}"
+echo "Copying ${UPSTREAM_IMAGE_NAME} to ${IMAGE_NAME}..."
+crane copy "${UPSTREAM_IMAGE_NAME}" "${IMAGE_NAME}"
```

Comment:
Since we already have `gcloud` can we just do:

```shell
gcloud container images add-tag --quiet "${UPSTREAM_IMAGE_NAME}" "${IMAGE_NAME}"
```

Reply:
I don't think that works for multi-plat, but I haven't tried it.